setup.py

```diff
@@ -29,7 +29,7 @@
     setup_requires=pytest_runner + wheel + ["setuptools_scm"],
     tests_require=["pytest>=2.8"],
     install_requires=[
-        "fonttools[ufo]>=4.38.0",
+        "fonttools[ufo]>4.38.0",
```

Comment:
let's wait for a fonttools release before bumping this (hopefully before end of this week)